exoatlas/observing/__init__.py

```diff
@@ -0,0 +1 @@
+from .transits import *
```

Comment:
The `from .transits import *` in here imports the code from your `transits.py` into the `observing` namespace, which then gets imported in `exoatlas/__init__.py` into the main package namespace. Taken all together, now we should be able to do `from exoatlas import find_observable_transits` and Python will look inside of `exoatlas/__init__.py` to see with `from .observing import *` that code should be imported from `exoatlas/observing/__init__.py`, and in there it will see `from .transits import *` which will import the actual code from your actual functions in `exoatlas/observing/transits.py`.

Reply:
I know this all feels a little convoluted, @MiriCaradonna , but you'll get used to it! It is *mostly* the case that `from package_a.directory_b.file_c import function_d` will look for code for a function `function_d` in a file located at `package_a/directory_b/file_c.py`. 

I think that all of the extra fanciness with `__init__.py` files inside of directories is effectively just enabling us to do a more direct (from the user's perspective) `from package_a import function_d`. It's probably not super necessarily except for complicate packages (which `exoatlas` is quickly growing into), but it might come in handy to be aware of it!

Reply:
https://docs.python.org/3/tutorial/modules.html talks about modules + directories + `__init__.py` files, possibly in more detail than you want ;-)